dynamo-db/cli-dynamoDB.sh

```diff
@@ -0,0 +1,69 @@
+aws dynamodb create-table \
```

Comment:
I think you are supposed to add db info to your serverless config via resources field
https://www.serverless.com/guides/dynamodb